server/api.py

```
@@ -77,17 +78,41 @@ async def create_chat_completion(request: ChatCompletionRequest):
+        raise
+
+
 @app.post("/v1/responses")
 async def create_chat_response(request: ResponsesRequest):
     """
     Create a response with openResponses format
     """
 
+    try:
+        ResponsesRequest.model_validate(request)
+    except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)